### PR TITLE
Update Cooking mod, Reforge Guide Skip, Unlimited zoom, Remove sunglare

### PR DESCRIPTION
**Pull request by kohupallintrax**
Update reforge guide skip pattern

Update cooking mod. The old function that was hooked doesn't appear to be called by the current client version. A new function is used with different arguments. A new method of setting the amounts has been developed. 

Saving of the last used values for cookingmod are now saved and loaded. 

Update Remove Sunglare

Update Unlimited zoom

### CODE REVIEW

File: Kanan/CookingMod.cpp [hunk 1]
```diff
@@ -16,25 +16,35 @@ namespace kanan {
...
...ddress = scan("client.exe", "48 89 ? ? ? 48 89 ? ? ? 56 57 41 ? 48 83 EC ? 0F 29 ? ? ? 44 8B ? 48 8B ? 45 33 ? 0F B6 ? ? ? ? ? E8 ? ? ? ? 44 8B ? ? ? ? ? 41 83 F9 ? 0F 84 ? ? ? ? 41 83 F9 ? 0F 8D ? ? ? ? 8B 85 ? ? ? ? 85 C0 0F 84 ? ? ? ? 44 3B ? 0F 86 ? ? ? ? 41 8B ? 2B C8 0F AF ? ? ? ? ? B8 ? ? ? ? F7 E1 C1 EA ? 44 8B ? ? ? ? ? 44 03 ? ? ? ? ? 44 03 ? ? ? ? ? 41 81 F8 ? ? ? ? 0F 83 ? ? ? ? 41 8D ? ? B9 ? ? ? ? 41 2B ? 3D ? ? ? ? 0F 46 ? 85 C9 0F 84 ? ? ? ? 42 01 ? ? ? ? ? ? 48 8B ? E8 ? ? ? ? 84 C0 0F 84 ? ? ? ? 48 8B ? ? ? ? ? E8 ? ? ? ? 48 8B ? E8 ? ? ? ? 48 8B ? 48 C7 44 24 70 ? ? ? ? 48 8D ? ? ? ? ? E8 ? ? ? ? 8B F8 8D 48 ? E8 ? ? ? ? 48 89 ? ? ? 48 8D ? ? 4C 8D ? ? ? ? ? 8D 57 ? E8 ? ? ? ? 48 8B ? ? ? 89 79 ? 41 B0 ? 48 8D ? ? ? 48 8B ? E8 ? ? ? ? 90 48 8B ? ? ? 48 85 ? 74 ? E8 ? ? ? ? 48 C7 44 24 70 ? ? ? ? 41 B8 ? ? ? ? 48 8D ? ? ? ? ? 48 8B ? ? ? ? ? E8 ? ? ? ? 48 8B ? ? ? ? ? 48 8B ? B2 ? FF 90 ? ? ? ? 44 89 ? ? ? ? ? 48 8B ? ? ? ? ? 48 8B ? ? ? ? ? E8 ? ? ? ? 8B BD ? ? ? ? 03 BD ? ? ? ? 03 BD ? ? ? ? 48 85 ? 0F 84 ? ? ? ? 8D 4F ? 81 F9 ? ? ? ? 0F 87 ? ? ? ? 48 8B ? ? ? ? ? 48 8B ? E8 ? ? ? ? 48 8B ? E8 ? ? ? ? 0F 28 ? 48 8B ? E8 ? ? ? ? 0F 28 ? F3 0F ? ? ? ? ? ? 48 8B");
```
Comment (cursey) [inline Kanan/CookingMod.cpp:47]: I'm a bit creeped out by this huge pattern. When I get some time I may revisit this pattern myself and look for ways of adding some sophistication to how this address is found.

**Review by cursey — APPROVED** [review]
LGTM

File: Kanan/CookingMod.cpp (hunk 1 above)
Comment (cursey) [reply Kanan/CookingMod.cpp:47]: No resolution necessary.

Comment (kohupallintrax) [reply Kanan/CookingMod.cpp:47]: Just to clarify - what was the reason for not needing a resolution?

**Pull request merged**